### PR TITLE
update README formatting; add missing dependency

### PR DESCRIPTION
**Pull request by ericcf**
- was missing minitest dependency
- updated rubocop dependency and config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Purple Robot Data Capture
 
-[![Build Status](https://travis-ci.org/cbitstech/purple_robot_data_capture_ruby.svg)](https://travis-ci.org/cbitstech/purple_robot_data_capture_ruby) [![security](https://hakiri.io/github/cbitstech/purple_robot_data_capture_ruby/master.svg)](https://hakiri.io/github/cbitstech/purple_robot_data_capture_ruby/master)
+[![Build Status](https://travis-ci.org/cbitstech/purple_robot_data_capture_ruby.svg)](https://travis-ci.org/cbitstech/purple_robot_data_capture_ruby)
```
Comment (michaelwehrley) [inline README.md:3]: not broken :+1:

File: README.md [hunk 2]
```diff
@@ -44,36 +53,47 @@ configure your Purple Robot HTTP Upload Endpoint to `http://your-ip:9292/`
...
     $ gem install rails
+    ...
 
 ### Create and configure the application
 
     $ rails new pr_endpoint
-    $ echo "gem 'purple_robot_data_capture_ruby', git: 'https://github.com/cbitstech/purple_robot_data_capture_ruby.git'" >> Gemfile
+    ...
+    $ echo "gem 'purple_robot_data_capture_ruby', \
+      git: 'https://github.com/cbitstech/purple_robot_data_capture_ruby.git'" \
```
Comment (michaelwehrley) [inline README.md:63]: Does it make sense to include the version?

```
$ echo "gem 'purple_robot_data_capture_ruby', git: 'https://github.com/cbitstech/purple_robot_data_capture_ruby.git', tag: 'X.X.X'" >> Gemfile
```

**Comment by michaelwehrley** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (ericcf) [inline README.md:63]: Yeah, I just didn't want to have to update the README every time the version changes.

**Pull request merged**